app/helpers/gcse_qualification_helper.rb

```diff
@@ -15,4 +15,24 @@ def hint_for_gcse_edit_grade(subject, qualification_type)
       t("gcse_edit_grade.hint.#{subject}.#{qualification_type}")
     end
   end
+
+  def grade_step_title(subject, qualification_type)
+    subject = subject.capitalize if subject == 'english'
```

Comment:
Please can we also capitalise Science and Maths here so it's consistent across the subjects

Reply:
@paulrobertlloyd ☝️ - can you confirm? Thought it was only English that was supposed to be capitalised?

Reply:
You're probably right. My grasp of the English language is tenuous at best.

Reply:
Yep, only English is capitalised.